charts/azuremonitor-containers/values.yaml

```diff
@@ -29,7 +29,7 @@ omsagent:
     agentVersion: "azure-mdsd-1.17.0"
     winAgentVersion: "0.0.0-0" # there is no base agent version for windows agent
 
-  # The priority used by the omsagent priority class for the daemonset pods
+  # The priority used by the ama-logs priority class for the daemonset pods
```

Comment:
Not a 100% sure about what this comment means, but should this be amalogs?

Reply:
i think this is fine since it is referring to the actual pod names rather than the internal value names